src/ploomber/cli/examples.py

```diff
@@ -20,6 +20,7 @@
 from pygments.formatters.terminal import TerminalFormatter
 from pygments.lexers.markup import MarkdownLexer
 from pygments import highlight
+from ploomber_cli.cli import _suggest_command
```

Comment:
let's declare a new `_suggest_example` on this file based on the implementation of `_suggest_command`, since the latter has things that we don't need here.

```python
def _suggest_examplename: str, options):
    if not name or name in options:
        return None

    name = name.lower()

    close_commands = get_close_matches(name, options)

    if close_commands:
        return close_commands[0]
    else:
        return None
```

Reply:
Sure. Have done the change